packages/dao/scripts/deriveNamehash.js

```diff
@@ -0,0 +1,6 @@
+const namehash = require("eth-ens-namehash");
```

Comment:
The only thing which is strange for me, do we really need this file?

Reply:
To call `namehash.hash()`

Reply:
Ah sorry, I understood your question now. The issue mentions that they don't know where the app ID comes from so they can't verify it, so they can verify that the app ID is the namehash of the app name (`api3voting.open.aragonpm.eth`). How to use it is in the README.